ur3e: #namespace
    task_and_robot_environment_name: 'UR3eBasicEnv-v0'
    ros_ws_abspath: "/home/cambel/ws_ur5"
    #qlearn parameters
    alpha: 0.1
    gamma: 0.7
    epsilon: 0.9
    epsilon_discount: 0.999
    nepisodes: 500
    nsteps: 10000

    ft_sensor: false
    driver: "/arm_controller/command"
    n_actions: 6
    slowness: 1
    init_pos: [0,0,0,0,0,0.]
    target_pos: [0.5,0.,0.]
    scale_pos: 100
    end_effector_points: [[0.,0.,0.]]
    distance_threshold: 0.05
    reward_type: 'continuous'
    dt: 0.002 

